### PR TITLE
fix: link to explore table when explore tokens banner clicked

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1448?atlOrigin=eyJpIjoiODBlNDJhMjIwMWQwNDdhZWIxOTg0NzYxMGI5YzlhNjQiLCJwIjoiaiJ9

### CODE REVIEW

File: src/components/Tokens/TokensBanner.tsx [hunk 1]
```diff
@@ -59,23 +60,30 @@ const Description = styled(Link)`
 export default function TokensBanner() {
   const theme = useTheme()
   const [showTokensPromoBanner, setShowTokensPromoBanner] = useShowTokensPromoBanner()
+  const navigate = useNavigate()
+  const chainName = useGlobalChainName().toLowerCase()
 
-  const closeBanner = () => {
-    setShowTokensPromoBanner(false)
+  const openExplorePage = () => {
```
Comment (JFrankfurt) [inline src/components/Tokens/TokensBanner.tsx:66]: nit: `open` kinda makes this sound like a popup

File: src/components/Tokens/TokensBanner.tsx [hunk 2]
```diff
@@ -59,23 +60,30 @@ const Description = styled(Link)`
...
+            setShowTokensPromoBanner(false)
+          }}
+          style={{ cursor: 'pointer' }}
+        />
       </Header>
 
-      <Description to={'/tokens'} onClick={closeBanner}>
-        Check out the new explore tab to discover and learn more
-      </Description>
+      <Description to={'/tokens'}>Check out the new explore tab to discover and learn more</Description>
```
Comment (JFrankfurt) [inline src/components/Tokens/TokensBanner.tsx:86]: add `<Trans />` wrapper

**Review by JFrankfurt — APPROVED** [review]
approved assuming non-nits are addressed

File: src/components/TopLevelModals/index.tsx [hunk 3]
```diff
@@ -24,6 +24,12 @@ export default function TopLevelModals() {
 
   useAccountRiskCheck(account)
   const open = Boolean(blockedAccountModalOpen && account)
+
+  console.log(
+    'info',
+    useTokensFlag() === TokensVariant.Enabled &&
+      (location.pathname.includes('/pool') || location.pathname.includes('/swap'))
+  )
```
[the remark this answers is not in the document]
Comment (lynnshaoyu) [reply src/components/TopLevelModals/index.tsx:32]: no thanks for catch.

File: src/components/Tokens/TokensBanner.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokensBanner.tsx:86]: ty for reminding!

File: src/components/Tokens/TokensBanner.tsx [hunk 4]
```diff
@@ -59,23 +60,30 @@ const Description = styled(Link)`
...
 
   return (
-    <PopupContainer show={showTokensPromoBanner}>
+    <PopupContainer show={showTokensPromoBanner} onClick={openExplorePage}>
       <Header>
-        <HeaderText to={'/tokens'} onClick={closeBanner}>
-          Explore Top Tokens
-        </HeaderText>
-        <X size={20} color={theme.textSecondary} onClick={closeBanner} style={{ cursor: 'pointer' }} />
+        <HeaderText to={'/tokens'}>Explore Top Tokens</HeaderText>
```
[the remark this answers is not in the document]
Comment (lynnshaoyu) [reply src/components/Tokens/TokensBanner.tsx:73]: done

File: src/components/Tokens/TokensBanner.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokensBanner.tsx:66]: true, changed to navigate

**Pull request merged**